policy/release/base_image_registries/base_image_registries.rego

```diff
@@ -125,6 +133,15 @@ _is_base_image(component) if {
 	count(base_image_properties) > 0
 }
 
+# spdx format
+_is_base_image(component) if {
```

Comment:
Just an idea: There's maybe not much value in having the same rule name as line 128 here. Consider naming it `_is_spdx_base_image` and `_is_cyclonedx_base_image`, then you could remove the two comments.

I guess the same applies for the two `_is_base_image_property` rules.

Reply:
That's a good idea. I'm updating `is_base_image`, the property function is separate so I'll leave that one for now.